modules/account-lib/src/coin/hbar/utils.ts

```diff
@@ -193,3 +193,11 @@ export function isValidMemo(memo: string): boolean {
   }
   return true;
 }
+
+/**
+ * @param buffer
+ * @param raw
+ */
+export function fromEd25519PublicKeyBuffer(buffer: Buffer, raw = false): string {
+  return Ed25519PublicKey.fromBytes(buffer).toString(raw);
```

Comment:
This is done by the `KeyPair` class in HBAR

Reply:
Ah yes. We can remove this then